main.go

```diff
@@ -38,6 +48,19 @@ type ConfigsModel struct {
 	ManualMerge      string
 }
 
+func init() {
```

Comment:
what's the point of init function? why you do not do it's functionality in main.go?